### PR TITLE
add mariabackup

### PR DESCRIPTION
**Pull request by mikegriffin**
Initial release

### CODE REVIEW

File: docs/source/provider_plugins/mariabackup.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+[mariabackup]
+____________
+
+**global-defaults** = <path> (default: /etc/my.cnf)
+
+    The MySQL configuration file for mariabackup to parse.  This is !include'd
+    into the my.cnf the mariabackup plugin generates
+
+**innobackupex** = <name> (default: innobackupex)
+
```
Comment (m00dawg) [inline docs/source/provider_plugins/mariabackup.rst]: Should be mariadbbackup here no?

File: config/backupsets/examples/mariabackup.conf [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+backups-to-keep = 1
+auto-purge-failures = yes
+purge-policy = after-backup
+estimated-size-factor = 1.0
+
+[mariabackup]
+global-defaults = /etc/my.cnf
+innobackupex = mariabackup
+# ibbackup = "" # no default
+stream = mbstream
```
Comment (m00dawg) [inline config/backupsets/examples/mariabackup.conf]: In digging throubh e-mails is this still correct or should it be xbstream?

**Comment by mikegriffin** [comment]
mariabackup ships eg /usr/bin/mbstream but only accepts --stream=xbstream. I assume this is "transitional" and that the command line args will be updated to --stream=mbstream, so I treat both as xbstream currently so that it will be easy to fix when the args are updated.
~mg

-----Original Message-----

**Comment by soulen3** [comment]
I've updated this for python3 and merged it into https://github.com/holland-backup/holland/pull/186

Thanks,
Steve

**Pull request closed without merging**